event-svc/src/event/service.rs

```diff
@@ -52,15 +54,15 @@ pub trait BlockStore {
     async fn block_data(&self, cid: &Cid) -> anyhow::Result<Option<Vec<u8>>>;
 }
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[derive(Debug, Clone, PartialEq, Eq)]
 pub enum DeliverableRequirement {
     /// Must be ordered immediately and is rejected if not currently deliverable. The appropriate setting
     /// for API writes as we cannot create an event without its history.
     Immediate,
     /// This will be ordered as soon as its dependencies are discovered. Can be written in the meantime
     /// and will consume memory tracking the event until it can be ordered. The appropriate setting for recon
     /// discovered events.
-    Asap,
+    Asap(String),
```

Comment:
I'm guessing the String here is the source DID string?  It feels pretty weird that Asap would include a source string but Lazy and Immediate don't.  I feel like we're conflating concepts here.

Reply:
Yes, we're now passing the source (`informant`) in separately.